### ISSUE TITLE
Add `aria-label` to Model Signature component

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2526]
Scope
---
To enhance accessibility, the Model Signature component should have the possibility to add more desriptive text to it.

Out of Scope
---
List topics that are not part of this task.

Notes  
---
- Slack: https://porschedev.slack.com/archives/C055XKEL4BA/p1682598935110939

Acceptance criteria
---
- [ ] Model Signature has `aria` prop with `aria-label`

Subtasks
---
- [ ] Task

**Comment by DerHannes85** [comment]
Can we have the respective model name as default?

**Comment by o-hook** [comment]
Closed due to the fact, that the Model Signature component has an `alt` test provided and no additional information needs to be exposed.
